.gitignore

```diff
@@ -16,6 +16,8 @@ pom.xml.asc
 .hg/
 .idea
 beat-link-trigger.iml
-
+.github/scripts/**/*
```

Comment:
These scripts are an important part of the project (they are part of how the installers for different platforms get built when changes are pushed to `main`) and should not be excluded from git.